cibuildwheel/macos.py

```diff
@@ -193,20 +193,23 @@ def install_pypy(tmp: Path, url: str) -> Path:
     return installation_path / "bin" / "pypy3"
 
 
+def can_use_uv(python_configuration: PythonConfiguration) -> bool:
+    conditions = (Version(python_configuration.version) >= Version("3.8"),)
+    return all(conditions)
```

Comment:
Why not just 
```
return Version(python_configuration.version) >= Version("3.8")
```

Reply:
Because I used the same layout as on Windows for consistency where using a conditions tuple is more readable.

Reply:
ok.